src/crawler.ts

```diff
@@ -1835,6 +1835,17 @@ self.__bx_behaviors.selectMainBehavior();
 
     await this.netIdle(page, logDetails);
 
+    logger.debug(
+      "Waiting for custom page load via behavior",
+      logDetails,
+      "behavior",
+    );
+    try {
+      await page.evaluate("self.__bx_behaviors.initialPageLoad();");
```

Comment:
Guess this already has its own timeout, yeah?

Reply:
Yes, it does, so wanted to avoid yet another timeout here, though could add. Of course, its also falls under the total page timeout time.

Reply:
Yup I think with the overall page timeout and the behavior timeout we're already well covered here :)